shopelectro/urls.py

```diff
@@ -132,3 +132,12 @@ def cache_page(arg):  # Ignore PyFlakesBear
         *static(settings.STATIC_URL, document_root=settings.STATIC_ROOT),
         *static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT),
     ]
+
+if settings.TEST_ENV or settings.DEBUG:
```

Comment:
Why did you declare 404 and 500 only for these environments?  We need them in production too, right?
```suggestion
if settings.TEST_ENV or not settings.DEBUG:
```

Reply:
@artemiy312 , noup. We need them only for dev and test envs.
This if is needed because Django hides real 404 and 500 pages on dev and test env to show traceback. So, i added them manually to for testing at test env and manual checking at dev env.
I'll add comment about it

Reply:
@duker33 500 page contains a lot of useful debugging info. I'd rather enable these views only for settings.TEST_ENV. What do you think?

Reply:
@artemiy312 , sorry, didn't noticed your comment before merge. Anyway, let's continue discussion there.

I don't disable debugging info for 500 errors. I just add new `/500` url that responses with real production version of 500 error page and 200 ok status